diff --git a/node_modules/react-native-tts/android/build.gradle b/node_modules/react-native-tts/android/build.gradle
index dbbe046..2c2f4c4 100644
--- a/node_modules/react-native-tts/android/build.gradle
+++ b/node_modules/react-native-tts/android/build.gradle
@@ -4,7 +4,7 @@ def safeExtGet(prop, fallback) {
 
 buildscript {
     repositories {
-        jcenter()
+        mavenCentral()
     }
 
     dependencies {
@@ -15,8 +15,8 @@ apply plugin: 'com.android.library'
 
 android {
+    namespace "net.no_mad.tts"
     compileSdkVersion safeExtGet('compileSdkVersion', 26)
-    buildToolsVersion safeExtGet('buildToolsVersion', '26.0.3')
 
     defaultConfig {
         minSdkVersion safeExtGet('minSdkVersion', 16)
diff --git a/node_modules/react-native-tts/android/src/main/AndroidManifest.xml b/node_modules/react-native-tts/android/src/main/AndroidManifest.xml
index a648498..57d6f85 100644
--- a/node_modules/react-native-tts/android/src/main/AndroidManifest.xml
+++ b/node_modules/react-native-tts/android/src/main/AndroidManifest.xml
@@ -1,2 +1,2 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="net.no_mad.tts">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
 </manifest>